src/select-menu/src/Option.js

```diff
@@ -37,38 +52,47 @@ const Option = memo(
       textProps = selectedProps
     }
 
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'Option',
+      emptyObject,
+      pseudoSelectors,
+      internalStyles
+    )
+
+    const { getRef, ...listBehaviorProps } = useListBehavior({
+      isSelectable,
+      isSelected,
+      disabled,
+      onSelect
+    })
+
+    const callbackRef = useMergedRef(getRef, forwardedRef)
+
     return (
-      <TableRow
-        isSelectable={isSelectable && !disabled}
-        isHighlighted={isHighlighted}
-        onSelect={onSelect}
-        onDeselect={onDeselect}
-        isSelected={isSelected}
+      <Pane
         style={style}
```

Comment:
Do we still need to support this? If we aren't touching `style` we can let it come through via `...rest`?